src/misc.c

```diff
@@ -1396,33 +1394,41 @@ cr_run_command(char **cmd, const char *working_dir, GError **err)
 }
 
 gboolean
-cr_cp(const char *src,
-      const char *dst,
-      cr_CpFlags flags,
-      const char *working_dir,
+cr_gio_cp(GFile *src,
+      GFile *dst,
+      GFileCopyFlags flags,
+      GCancellable *cancellable,
       GError **err)
```

Comment:
This deletes the cr_cp function and replaces it with cr_gio_cp, but does not implement an additional copy of cr_cp (potentially as a thin wrapper over cr_gio_cp). However, misc.h only appended a new function, so the header says cr_cp exists but no function is ever actually compiled.

This seems to go counter to the above discussion... did you forget to re-add that in?

The update diff is https://github.com/rpm-software-management/createrepo_c/compare/7f99f9a675a58ba4cbf76e1a86b3c30301bcc13d..6a9a871abcfd2685a724f5ed1428d4de56ff5bb6 and shows an re-inclusion in misc.h but not misc.c

Reply:
Also as discussed elsewhere, `__attribute((deprecated("please use cr_gio_cp instead")))` or something to that general effect would be apropos. :)